app/assets/stylesheets/helpers/_layouts.scss

```diff
@@ -1,8 +1,8 @@
 .wrapper {
-  border-bottom: 10px solid $govuk-blue;
-
   .content {
+    @extend %contain-floats;
+
     display: block;
-    @include outer-block;
+    padding: 0 15px;
```

Comment:
We should be using the measurement variables here I think

https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_measurements.scss